pkg/builder/builder.go

```diff
@@ -354,9 +355,10 @@ func (b generalBuilder) build(ctx context.Context, pw progresswriter.Writer) err
 
 func (b generalBuilder) checkIfNeedBuild(ctx context.Context) bool {
 	depsFiles := []string{
-		b.PubKeyPath,
 		b.ConfigFilePath,
 	}
+	depsFiles = getDepsFiles(depsFiles)
+	fmt.Println(depsFiles)
```

Comment:
Please remove the fmt.Print statement here.

Reply:
OK, maybe I should fix my local go-lint, It is very strange.